django/otto/fixtures/cost_types.yaml

```diff
@@ -168,5 +168,5 @@
     description_fr: Appels API au modèle de déploiement global Azure OpenAI GPT-4o-mini (sortie)
     unit_name: tokens
     unit_name_fr: jetons
-    unit_cost: 0.0006
-    unit_quantity: 1000
\ No newline at end of file
+    unit_cost: 0.6
```

Comment:
Why has everything been multiplied by 1000?

Reply:
This is how the Azure pricing page shows the costs now.